apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -473,21 +473,15 @@ handle_request_('GetPendingTransactions', _Req, _Context) ->
     {200, [], #{transactions => [aetx_sign:serialize_for_client_pending(T) || T <- Txs]}};
 
 handle_request_('GetPeers', _Req, _Context) ->
-    case aeu_env:user_config_or_env([<<"http">>, <<"debug">>],
```

Comment:
Is "http > debug" in user config schema a leftover?

Reply:
Marked as deprecated, see https://github.com/aeternity/epoch/pull/1901#discussion_r243129204